openmc/universe.py

```diff
@@ -360,6 +360,10 @@ def plot(self, origin=None, width=None, pixels=40000,
         outline : bool
             Whether outlines between color boundaries should be drawn
 
+            .. versionadded:: 0.13.4
+        axis_units : {'km', 'm', 'cm', 'mm'}
+            Units used on the plot axis, the default is cm
```

Comment:
No need to mention default since it is in the method signature
```suggestion
            Units used on the plot axis
```